cmd/kube-apiserver/app/server.go

```diff
@@ -123,7 +125,7 @@ func Run(s *options.ServerRunOptions) error {
 
 	// Setup tunneler if needed
 	var tunneler genericapiserver.Tunneler
-	var proxyDialerFn apiserver.ProxyDialerFunc
+	var proxyDialerFn func(network, addr string) (net.Conn, error)
```

Comment:
In fact, the func type is used at several places, the alias isn't. Will factor that out.